iceoryx2-cli/lib/src/config_descriptions.rs

```diff
@@ -0,0 +1,389 @@
+                    description: "Specifies the path for service-related files under `global.root-path`.",
+                },
+                CliEntry {
+                    key: "global.service.data-segment-suffix",
+                    value_type: "string",
+                    default: format!("\"{}\"", config.global.service.data_segment_suffix),
+                    description: "Suffix added to the ports's data segment.",
+                },
+                CliEntry {
+                    key: "global.service.static-config-storage-suffix",
+                    value_type: "string",
+                    default: format!("\"{}\"", config.global.service.static_config_storage_suffix),
+                    description: "Suffix for static service configuration files.",
+                },
+                CliEntry {
+                    key: "global.service.dynamic-config-storage-suffix",
+                    value_type: "string",
+                    default: format!("\"{}\"", config.global.service.dynamic_config_storage_suffix),
+                    description: "Suffix for dynamic service configuration files.",
+                },
+                CliEntry {
+                    key: "global.service.event-connection-suffix",
+                    value_type: "string",
+                    default: format!("\"{}\"", config.global.service.event_connection_suffix),
+                    description: "Suffix for event channel.",
+                },
+                CliEntry {
+                    key: "global.service.connection-suffix",
+                    value_type: "string",
+                    default: format!("\"{}\"", config.global.service.connection_suffix),
+                    description: "Suffix for one-to-one connections.",
+                },
+                CliEntry {
+                    key: "global.service.blackboard-mgmt-suffix",
+                    value_type: "string",
+                    default: format!("\"{}\"", config.global.service.blackboard_mgmt_suffix),
+                    description: "The suffix of the blackboard management data segment.",
+                },
+                CliEntry {
+                    key: "global.service.blackboard-data-suffix",
+                    value_type: "string",
+                    default: format!("\"{}\"", config.global.service.blackboard_data_suffix),
+                    description: "The suffix of the blackboard payload data segment.",
+                },
+            ],
+        },
+        CliSection{
+            name: "Global: Service Creation Timeout",
+            entries: vec![
+                CliEntry {
+                    key: "global.service.creation-timeout.secs",
+                    value_type: "int",
+                    default: config.global.service.creation_timeout.as_secs().to_string(),
+                    description: "Maximum time for service setup in seconds. Uncreated services after this are marked as stalled.",
```

Comment:
It seems this works only when `nanos` is also set and vice versa. This should be visible in the description. Could you please add something like `Attention: if 'secs' is set, 'nanos' must also be set`. Feel free to choose something else if you have a good idea.